SocialFilter/sections/specific.txt

```diff
@@ -99,6 +99,7 @@ karapaia.com##div[style^="margin:1.5em 0 0.7em;font-size:14px;font-weight:bold;l
 back-in-ussr.com###divShareTopicFoother
 octagon.media##.art_footer > ul
 yamal-media.ru##div[class^="share_Share__"]
+diariodocomercio.com.br##div[id^="eapps-social-share-buttons"]
```

Comment:
You should also remove this
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/33488229/141102335-5b2fb784-853e-4565-a95b-3d8da07e9b8f.png)

</details><br/>